xml/System.Text/Rune.xml

```diff
@@ -573,8 +589,8 @@ This method is very similar to <xref:System.Text.Rune.DecodeFromUtf8(System.Read
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Returns <see langword="true" /> if and only if this scalar value is ASCII ([ U+0000..U+007F ]) and therefore representable by a single UTF-8 code unit.</summary>
-        <value>To be added.</value>
+        <summary>Returns <see langword="true" /> if and only if this scalar value is ASCII ([ U+0000..U+007F ]) and therefore representable by a single UTF-8 code unit; <see langword="false" /> otherwise.</summary>
+        <value>A boolean indicating whether this scalar value is ASCII or not.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if the scalar value associated is within the ASCII encoding range ([ U+0000..U+007F ]) and therefore representable by a single UTF-8 code unit; otherwise, <see langword="false" />.</value>
```